src/transport/manager/address.rs

```diff
@@ -222,6 +232,11 @@ impl AddressStore {
     /// If the address is not in the store, it will be inserted.
     /// Otherwise, the score and connection ID will be updated.
     pub fn insert(&mut self, record: AddressRecord) {
+        if record.score == scores::ADDRESS_FAILURE {
```

Comment:
```suggestion
        if record.score <= scores::UNRECOVERABLE_FAILURES {
```
This way it will look less like some magic number using.